### DISCUSSION TITLE
Can you help me with this error?

### DISCUSSION THREAD

**Discussion by REDLIGHTFILMS** [uber/react-map-gl#1423] [Q&A]
Hi, after updating react-map-gl from 5.2.7 to 5.3.11 I suddenly got this error. I use Typescript version 3.9.5 

How to fix the error?

My Error: TypeScript error in /Users/redlightfilms/Desktop/adminapp/node_modules/react-map-gl/src/components/interactive-map.d.ts(12,12): ',' expected.  TS1005

**Comment by Pessimistress** [comment]
I believe [labeled tuple](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#labeled-tuple-elements) is a TypeScript 4.0 feature:

https://github.com/visgl/react-map-gl/blob/master/src/components/interactive-map.d.ts#L12

@Jamie5

**Reply by Jamie5** [reply]
Hmm yes that seems correct. Would the fix then be to drop the labels until we're ready to drop pre-4.0 support?

**Reply by Pessimistress** [reply]
I'm not sure how many users are still on older versions. I would advocate for better compatibility, since the tradeoff seems pretty small.

**Reply by Jamie5** [reply]
Ok, #1426 should fix it

**Comment by Pessimistress** [comment]
This has been fixed in 5.3.12
